aii-opennebula/src/main/perl/opennebula.pod

```diff
@@ -96,8 +99,6 @@ All these paths are relative to /system/aii/hooks/remove
 
 Stop running VM
 
-=over 4
```

Comment:
It is included with the components in configuration-modules-core, but not here it seems. @alvarosimon you can add the file pod-syntax.t to the test dir with contents 

```
use Test::More;
use Test::Pod;

my @dirs = qw(target/lib/perl target/doc/pod);
all_pod_files_ok(all_pod_files(@dirs));
```